lib/DDG/Spice/Twitter.pm

```diff
@@ -5,16 +5,21 @@ use strict;
 use DDG::Spice;
 
 spice to => 'https://duckduckgo.com/tw.js?user=$1&callback={{callback}}&current=1';
-triggers query => qr/^(?:twitter\s)?@([a-z0-9_]+)$|^twitter\s([a-z0-9_]+)$/i;
+spice proxy_cache_valid => '200 1h';
```

Comment:
@nalinbhardwaj I don't think we can do this -- we're likely going to go over our API call limit if we reduce our caching time.

Reply:
I suppose this PR is redundant then. I'll just be closing it.

Reply:
@nalinbhardwaj I still think there's merit in the triggering changes and showing the follower count. It's just that the follower count will update daily -- I don't think anyone needs a near realtime number for followers?

Reply:
@moollaza didn't you say you made internal changes to improve triggering?
Those don't cover the ones I did? If not, I'll change this to 1 day if that's fine?

Reply:
> didn't you say you made internal changes to improve triggering?

@nalinbhardwaj we made some, but they don't include what you've done here.

> Those don't cover the ones I did? If not, I'll change this to 1 day if that's fine?

The default is 24 hours, so no changes to the caching are necessary 👍

Reply:
@moollaza done, sorry for all the delays